cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgstautau.json

```diff
@@ -27,7 +27,7 @@
       {
         "checksum": "adler32:937a3ec4",
         "size": 1012110,
-        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/HiggsTauTauNanoAODOutreachAnalysis/HiggsTauTauNanoAODOutreachAnalysis-0.1.zip"
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/HiggsTauTauNanoAODOutreachAnalysis/HiggsTauTauNanoAODOutreachAnalysis-1.0.zip"
```

Comment:
I'll grab the new tarball and I see that `skim.cxx` and `histograms.py` have also changed so I re-download these to EOS and update checksums etc accordingly.

Reply:
Perfect!